pkg/operator/controllers/workaround/workaround_controller.go

```diff
@@ -51,6 +52,16 @@ func NewReconciler(log *logrus.Entry, kubernetescli kubernetes.Interface, config
 
 // Reconcile makes sure that the workarounds are applied or removed as per the OpenShift version.
 func (r *WorkaroundReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
+	instance, err := r.arocli.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
+	if err != nil {
+		r.log.Error(err)
```

Comment:
Why we log here, but not in other places? I think controller-runtime will log errors for us, so this logging is not necessary.